pkgs/by-name/tq/tqftpserv/package.nix

```diff
@@ -12,13 +12,15 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "tqftpserv";
-  version = "1.1";
+  # Use unstable for fixed systemd service
```

Comment:
Are you actually able to use the upstream systemd service for this unchanged in `systemd.packages`?

Reply:
Yes, after these updates the upstream services work fine for me in systemd.packages.

Perviously, the services depended on qrtr-ns.service which no longer was needed because qrtr-ns has been moved to the kernel.